LanguageFeatures/Wildcards/unchanged_A05_t07.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Other declarations: top-level variables, top-level function
+/// names, type names, member names, etc. are unchanged. They can be named `_`
+/// as they are today.
+///
+/// @description Checks that it is a compile-time error to declare more than one
+/// member named `_`. Test a static getter vs other class members conflict.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+class C1 {
+  static int get _ => 1;
+  static int _ = 1;
+//           ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C2 {
+  static int get _ => 1;
+  static int _() => 2;
+//           ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C3 {
+  static int get _ => 1;
+  static int get _ => 3;
+//               ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C4 {
+  static int get _ => 1;
+  static void set _(int v) {} // Ok
+}
+
+class C5 {
```

Comment:
Same here: We could change a couple of these cases such that the instance member is in a superinterface.